kaspad/src/daemon.rs

```diff
@@ -30,6 +30,15 @@ use kaspa_perf_monitor::builder::Builder as PerfMonitorBuilder;
 use kaspa_utxoindex::{api::UtxoIndexProxy, UtxoIndex};
 use kaspa_wrpc_server::service::{Options as WrpcServerOptions, ServerCounters as WrpcServerCounters, WrpcEncoding, WrpcService};
 
+/// Desired soft FD limit that needs to be configured
+/// for the kaspad process.
+pub const DESIRED_DAEMON_SOFT_FD_LIMIT: u64 = 16 * 1024;
+/// Minimum acceptable soft FD limit for the kaspad
+/// process. (Rusty Kaspa will operate with the minimal
+/// acceptable limit of `1024`, but a setting below
+/// this value may impact the database performance).
+pub const MINIMUM_DAEMON_SOFT_FD_LIMIT: u64 = 4096;
```

Comment:
The comment says `1024` but this value is `4096`. Why the difference? And if it really is `4096` can we express it as `4 * 1024` to be consistent with the above?

Reply:
I made the comment say 4096 and the declaration to be 4*1024.  I use full numbers below 10k at which point it becomes hard to read.

These values perhaps should be changed down the road.  I am defining 4096 from my local observations on T10 where a typical rusty kaspa consumption without limit currently ends up being slightly over 3k.  It is possible that it will be more on the current mainnet.